master/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml

```diff
@@ -184,6 +185,9 @@ spec:
             # Disable IPv6 on Kubernetes.
             - name: FELIX_IPV6SUPPORT
               value: "false"
+            # Set MTU for tunnel device
+            - name: FELIX_IPINIPMTU
+              value: 1440
```

Comment:
?? You said yesterday that the felix code listed 1440.
I just double checked, the code has 1440.  https://github.com/projectcalico/felix/blob/2bb4220c59210c94934ce13602085da699def2f5/config/config_params.go#L131